android/app/src/main/java/updated/mysterium/vpn/database/entity/NodeEntity.kt

```diff
@@ -19,9 +19,10 @@ data class NodeEntity(
     @ColumnInfo(name = "price_per_second") val pricePerSecond: Double,
     @ColumnInfo(name = "price_per_byte") val pricePerByte: Double,
     @ColumnInfo(name = "quality_level") var qualityLevel: Int,
+    @ColumnInfo(name = "is_saved") var isSaved: Int,
```

Comment:
Why it's not boolean?

Reply:
I can change it to Boolean, but query will still the same, because SQL not support boolean data type, right?
Like this: @Query("SELECT * FROM NodeEntity WHERE is_saved is 1") even if "is_saved" is Boolean

Reply:
Fix